lib/newgistics/requests/post_inbound_return.rb

```diff
@@ -15,8 +16,16 @@ def body
         xml_builder.to_xml
       end
 
+      def perform
+        Newgistics.api.post(self, response_handler.new(inbound_return))
```

Comment:
Can we assume that `response_handler` is already an instance of the desired class? That way if a developer wants to inject a custom response handler they can take care of instantiation. I.e the initializer of their custom handler can have a different signature than ours